docs/index.rst

```diff
@@ -65,6 +65,19 @@ that stream specifically on those machines.
            the `Livepatch documentation`_, or learn
            :doc:`how to enable it<howtoguides/enable_livepatch>` with ``pro``.
 
+       * **Landscape**
+
+         `Landscape`_ is the leading management and administration tool for
+         Ubuntu. It can manage up to 40,000 Ubuntu machines with a single
+         interface and can automate security patching as well as hardening
+         and compliance. Landscape can be `self-hosted`_ or you can use Canonical's
+         `Landscape SaaS`_ offering. You can register a machine with Landscape
+         using ``pro enable landscape``.
```

Comment:
Do we need to include the command to enable here? It sort of negates the need to include the guide immediately after it

Reply:
Right, I'll just remove this last sentence and keep the link to the guide after